javascript-constructors/student.js

```diff
@@ -0,0 +1 @@
+/* exported Student */
```

Comment:
@OSAdams Please define the `Student` constructor:

---

![image](https://user-images.githubusercontent.com/16143771/167743063-32fd9d32-54a8-4cd7-af50-51d87919f08d.png)

Reply:
OOF. My bad. Fixing now.